php-practice/section5.php

```diff
@@ -320,3 +338,25 @@ public function send() {
  *   'cost' => 3.99
  * ];
  */
+
+class ShoppingCart {
+  private $item;
+  private $shoppingCart;
+
+  $item = [
+    'name' => 'Pringles',
+    'cost' => 3.99
+  ];
+
+  public function addItem() {
+    $shoppingCart =  [];
+    $shoppingCart[] = array_push($item);
+  }
+
+  public function getItemCount() {
+    return $shoppingCart;
+  }
```

Comment:
This method returns `null` because `$shoppingCart` isn't defined in this function. You meant to do `$this->shoppingCart`, but that won't return the right thing either -- it will just return the array of items in the shopping cart.

What you want to do is use a function I realized I hadn't introduced you to -- `count()` -- Check it out! http://php.net/manual/en/function.count.php

See if you can use the docs to figure out how to fix this method. It should return an **integer**.